### PR TITLE
Fix code highlighting in input/output live spec

### PR DESCRIPTION
**Pull request by shiyunon**
### Summary

Simple fix of code highlighting in input/output section in the live spec.

### Details and comments

Fix code highlighting in input/output section in the live spec by using `:force:`.

### CODE REVIEW

File: source/language/directives.rst [hunk 1]
```diff
@@ -8,3 +8,103 @@ that give additional information to compiler passes and the target
 system or simulator. Ideally the meaning of the program does not change
 if some or all of the directives are ignored, so they can be interpreted
 at the discretion of the consuming process.
+<<<<<<< Updated upstream
```
Comment (zachschoenfeld33) [inline source/language/directives.rst:11]: This looks like a lingering merge conflict?

Comment (shiyunon) [reply source/language/directives.rst]: Okay, fixed.

**Comment by stevenheidel** [comment]
Thanks @shiyunon - this looks good once the merge conflicts are addressed

**Comment by shiyunon** [comment]
> Thanks @shiyunon - this looks good once the merge conflicts are addressed

Fixed in the newest commit.

**Review by blakejohnson — DISMISSED** [review]

**Review by blakejohnson — APPROVED** [review]

**Pull request merged**